components/GithubUsernameForm.tsx

```
@@ -71,13 +215,109 @@ export default function GithubUsernameForm() {
+                  </div>
+                </div>
+                <div className="space-y-1 max-h-56 overflow-y-auto pr-1 overscroll-contain">
+                  {repos.map((repo) => (
+                    <div key={repo} className="bg-background p-2 rounded border hover:bg-accent transition-colors">
+                      <a
+                        href={`https://github.com/${repo}`}
+                        target="_blank"
+                        rel="noopener noreferrer"
+                        className="text-sm text-primary hover:text-primary/80 hover:underline flex items-center"
+                      >
+                        <svg xmlns="http://www.w3.org/2000/svg" className="h-4 w-4 mr-1 text-muted-foreground flex-shrink-0" fill="none" viewBox="0 0 24 24" stroke="currentColor">
```

Biome (2.1.2)

[error] Alternative text title element cannot be empty

For accessibility purposes, SVGs should have an alternative text, provided via title element. If the svg element has role="img", you should add the aria-label or aria-labelledby attribute.

(lint/a11y/noSvgWithoutTitle)